.github/workflows/release-selfhost.yml

```diff
@@ -54,9 +54,6 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_API_KEY }}
           SELFHOST_TAG: latest
 
-      - name: Install Pro
-        run: yarn install:pro $BRANCH $BASE_BRANCH
```

Comment:
I think we can remove `BRANCH` and `BASE_BRANCH` also, for both pipelines. They are incorrect for this type of job as they're pull request based